eisenhower-matrix/src/main/webapp/script.js

```diff
@@ -45,52 +45,22 @@ function checkTitle () {
 
 // add each posted task to DOM; append it to #fetched-content div
 async function getTasks() {
-  let task_div = document.createElement("div");
 
-  const response = await fetch("/add-task");
-  const text = await response.text();
-
-  var all_tasks = text.split("\n");
-  all_tasks = all_tasks.filter((x) => x.length > 0);
-
-  all_tasks.forEach(function (val, idx, arr) {
-    arr[idx] = JSON.parse(val);
-  });
+    if (document.getElementById("tasks")) {
+        var tasks = document.getElementById("tasks");
+                tasks.parentNode.removeChild(tasks);
+    }
+    let task_div = document.createElement("div");
+    task_div.id = "tasks";
+    const response = await fetch("/add-task");
```

Comment:
You can use promises here and save 1 line, something like: `const response = fetch("/add-task").then(() => response.text() )`. It's more the JS way.